roles/builder/users.nix

```diff
@@ -22,6 +22,11 @@ let
       trusted = true;
       keys = ./keys/jtojnar;
     };
+
+    winter = {
+      trusted = true;
```

Comment:
@Mic92 trusted user basically gives root to the machine. Are we comfortable with that?

Reply:
I followed in the footsteps of the aarch64 builder. If you'd like me to negate that, let me know.

Reply:
I had a chat with @andir. Let's go one step back. @winterqt maybe you can give us feedback on how not beeing a trusted user affects your workflow. It might be not possible to use our server as a remote builder in this case?

Reply:
I do not believe that I need to be a trusted user to make use of this server, as most of my needs are building Nixpkgs related things such as NixOS tests. I will let you know if that changes though, if that would help.

Reply:
@Mic92 Nevermind:
```
> error: you are not privileged to build input-addressed derivations
```

This was when building a NixOS configuration (for tests). I can use the aarch64 builder for this, so it's not an issue, but figured I'd let you know.

Reply:
Oh. This is a bummer. Thanks for letting us know.

Reply:
Nope.

```
$ nix --store ./s build -f thelounge.nix --argstr system x86_64-linux -L --show-trace
error: creating directory '/s': Read-only file system
error: unexpected EOF reading a line

       … while reading the response from the build hook
```